plutus-scb/plutus-scb.cabal

```diff
@@ -31,23 +37,98 @@ common lang
     ghc-options: -Wall -Wnoncanonical-monad-instances
                  -Wincomplete-uni-patterns -Wincomplete-record-updates
                  -Wredundant-constraints -Widentities
+                 -- See Plutus Tx readme
+                 -fobject-code -fno-ignore-interface-pragmas -fno-omit-interface-pragmas
+    if flag(defer-plugin-errors)
+        ghc-options: -fplugin-opt Language.PlutusTx.Plugin:defer-errors
 
 library
     import: lang
     exposed-modules:
-        Plutus.SCB.Events.Contract
+        Plutus.SCB.Arbitrary
+        Plutus.SCB.Core
         Plutus.SCB.Events
+        Plutus.SCB.Relation
+        Plutus.SCB.Command
+        Plutus.SCB.Query
+    other-modules:
+        Plutus.SCB.Utils
+        Plutus.SCB.Events.Contract
+        Plutus.SCB.Events.Mock
     hs-source-dirs: src
     build-depends:
         plutus-wallet-api -any,
         plutus-emulator -any,
         plutus-tx -any,
+        plutus-tx-plugin -any,
         plutus-contract -any,
-        iots-export -any,
+        iots-export -any
     build-depends:
```

Comment:
why do we have two `build-depends` stanzas anyway? :thinking:

Reply:
That's a question for @j-mueller. I'd guessed it was to separate out plutus packages from the other dependencies.

Reply:
Yeah I usually separate the "friendly dependencies" from the other packages to make them easier to find

Reply:
I like it. :-)